src/components/services/Spa.vue

```diff
@@ -1,39 +1,226 @@
 <template>
   <div class="spa-home">
-    <h1>Spa Home</h1>
-    <h3 class="greetings">{{ msg }}</h3>
+
+    <div>
+      <CustomHeader title="My Concierge" subtitle="Spa" />
+    </div>
+
+    <div class="navbar-container">
+      <nav>
+        <ul>
+            <li><a href="#">Home</a></li>
+            <li class="sep">|</li>
+            <li><a href="#">Calendar</a></li>
+            <li class="sep">|</li>
+            <li><a href="#">Services</a></li>
+        </ul>
+      </nav>
+    </div>
+
+    
+    <div class="main-container">
+
+    <div class="col1">
```

Comment:
Check the indentation in this whole file. A child should be indented once and only once compared to the parent. This is true in the \<template\>, <script> and <style> sections.